content/product/product_management/index.md

```diff
@@ -53,6 +53,34 @@ Part of the product data is a (internal-only) list of [what's supported](https:/
 
 Launch tier levels (L1, L2, and L3) are also an important term to be aware of, and the definitions as well as process/source of truth are documented on the [marketing launch page](../../marketing/product-marketing/marketing_launch_tiers.md).
 
+## Collaboration
+
+Product Management tends to collaborate with a lot of different groups. Beyond our shared [work as a team value](../../company/values.md#work-as-a-team), there are some teams where collaboration is especially important or can be unclear so we have documented high level practices around how we work together.
+
+### Product Management & Marketing
+
+Product Managemetn and Marketing must work close together to successfully create and launch products. Because of this, it's important to be clear on what the shared and unique responsibilities are within the working relationship.
+
+TODO add link to source of this info
```

Comment:
@serinadawn do you have a link or something where we can credit the inspiration for this?

Reply:
@serinadawn I'll close this thread, but if you think of something feel free to add in a follow-up PR.

```suggestion
```